must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -250,12 +250,25 @@ for ns in $namespaces; do
         blockpools=$(timeout 60 oc get cephblockpools.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
         for bp in $blockpools; do
```

Comment:
Add `printf "Collecting image and snap info for images in: %s\n" "${bp}" >> "${COMMAND_OUTPUT_FILE}"` here.